index.js

```diff
@@ -11,13 +11,14 @@ const resolvers = require('./src/resolvers.js')
 const DB = require('./src/db/db')
 const IncidentStorage = require('./src/db/incidentStorage')
 const OctokitHelper = require('./src/git/ocotokitHelper')
+const Frontmatter = require('.src/db/frontmatter.js')
```

Comment:
Warum wird die frontmatter Klasse in der index.js eingebunden und nicht direkt in incidentStorage.js?
Ist der Code so nicht komplizierter und IncidentStorage von der index.js abhänig?

Reply:
IncidentStorage wird dadurch eigentlich sogar unabhängiger vom Rest, weil nicht oben irgendein spezifisches Modul von einer fixen Stelle eingebunden wird, sondern nur ein ein object erwartet wird, dass die beiden Methoden für Frontmatter bereitstellt. Die Anwendung wird allerdings nie so groß werden, dass das bei uns zum Problem werden würde, von daher könnte man das hier auch anders lösen. In großen Projekten würde man vermutlich Dependency Injection benutzen.